schema/meltano.schema.json

```diff
@@ -762,6 +776,45 @@
         }
       }
     },
+    "jobs": {
+      "type": "object",
+      "required": [
+        "name",
+        "tasks"
+      ],
+      "description": "https://docs.meltano.com/reference/command-line-interface#job",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "This jobs unique name",
+          "examples": ["gitlab-to-jsonl"]
+        },
+        "tasks": {
```

Comment:
this is actually lifted straight from the validation we do when adding a task via the cli: https://github.com/meltano/meltano/blob/main/src/meltano/core/task_sets.py#L17